src/server/bg_services/replication_server.js

```diff
@@ -85,7 +85,7 @@ async function copy_objects_mixed_types(req) {
             try {
                 await noobaa_con.copyObject(params).promise();
                 copy_res.num_of_objects += 1;
-                copy_res.size_of_objects += keys_diff_map[key][0].Size;
+                copy_res.size_of_objects += keys_diff_map[key][0].ContentLength;
```

Comment:
In case of `(keys_diff_map[key].length === 1)` we should probably check if it is version or not.
If it is version, we should use `Size`
if it is not then we can use `ContentLength`
when we do not have version we are doing `head_object`: 
https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html

We can check if the key object here has`VersionId`